molecule/kotlin-js-store/yarn.lock

```diff
@@ -55,6 +55,13 @@ brace-expansion@^1.1.7:
     balanced-match "^1.0.0"
     concat-map "0.0.1"
 
+brace-expansion@^2.0.1:
```

Comment:
What's going on, here? Why are dependencies being bumped?

Reply:
Missed from the Kotlin 1.7 upgrade. Need to add a GitHub action that fails on dirty files in the repo after a build.